ui/src/config/constants.tsx

```diff
@@ -1,9 +1,9 @@
 window.config = window.config || {
   API_URL: 'no API_URL  set',
-  GH_CLIENT_ID: 'no GH_CLIENT_ID set'
+  AUTH_BASE_URL: 'no AUTH_BASE_URL set'
```

Comment:
don't we need to add `REDIRECT_URI` here as well?

Reply:
not needed, but it's good to add and I will add for that